core/src/main/java/com/orientechnologies/orient/core/storage/impl/local/OAbstractPaginatedStorage.java

```diff
@@ -107,6 +110,42 @@
   private static final Comparator<ORecordOperation> COMMIT_RECORD_OPERATION_COMPARATOR = Comparator
       .comparing(o -> o.getRecord().getIdentity());
 
+  private static volatile DataOutputStream journaledStream = null;
```

Comment:
Can be final instead of volatile, which should improve performance a bit.

Reply:
That will block OrientDB instance during static initialization and crash tests server will be unable to report about its successful startup. I will try change this to final, but let's postpone that until crash tests refactoring.